documentation/src/main/asciidoc/user_guide/server_protocols_hotrod_transaction.adoc

```diff
@@ -0,0 +1,176 @@
+:tx: link:{javaeedocroot}/javax/transaction/Transaction.html[Transaction]
+:tm: link:{javaeedocroot}/javax/transaction/TransactionManager.html[TransactionManager]
+:sync: link:{javaeedocroot}/javax/transaction/Synchronization.html[Synchronization]
+:xa: link:{javaeedocroot}/javax/transaction/xa/XAResource.html[XAResource]
+:rcm: link:{javadocroot}/org/infinispan/client/hotrod/RemoteCacheManager.html[RemoteCacheManager]
+:rc: link:{javadocroot}/org/infinispan/client/hotrod/RemoteCache.html[RemoteCache]
+:tx-mode: link:{javadocroot}/org/infinispan/client/hotrod/configuration/TransactionMode.html[TransactionMode]
+:cb: link:{javadocroot}/org/infinispan/client/hotrod/configuration/ConfigurationBuilder.html[ConfigurationBuilder]
+:gtml: link:{javadocroot}/org/infinispan/client/hotrod/transaction/lookup/GenericTransactionManagerLookup.html[GenericTransactionManagerLookup]
+:rtml: link:{javadocroot}/org/infinispan/client/hotrod/transaction/lookup/RemoteTransactionManagerLookup.html[RemoteTransactionManagerLookup]
+:rtm: link:{javadocroot}/org/infinispan/client/hotrod/transaction/manager/RemoteTransactionManager.html[RemoteTransactionManager]
+:trc: link:{javadocroot}/org/infinispan/client/hotrod/TransactionalRemoteCache.html[TransactionalRemoteCache]
+
+==== Hot Rod Transaction
+
+The following article provide details about how to configure the Hot Rod client to be a participant in a
```

Comment:
This section provides details about how to configure and use the Hot Rod client to be a participant in a JTA {tx}.